### ISSUE TITLE
AuthorInfo

### ISSUE THREAD

**Issue by ikovac** [ukalwa/scholarly#39]
Hello scholarly developers,
will it be possible to get Author's info such as hindex, institution etc ? I found interface IAuthorInfo but it's never used. Will it be a next feature?

`export interface IAuthorInfo {
  allCitations: number;
  hIndex: number;
  i10Index: number;
  interests?: string[];
}
`

**Comment by ukalwa** [comment]
Yes, that is something in the works. Got distracted with other things and didn't spend a lot of time on it but I can bump up its priority and work on it 😄
